sdk/servicebus/service-bus/src/queueClient.ts

```diff
@@ -217,7 +234,11 @@ export class QueueClient implements Client {
    *
    * @param fromSequenceNumber The sequence number from where to read the message.
    * @param [maxMessageCount] The maximum number of messages to peek. Default value `1`.
-   * @returns Promise<ReceivedSBMessage[]>
+   * @returns Promise<ReceivedMessageInfo[]>
+   * @throws {Error} Thrown if the underlying connection or client is already closed.
```

Comment:
Missing `TypeError` for
- missing parameter for `fromSequenceNumber`
- value provided for `peekBySequenceNumber` is not a Long
- value provided for `maxMessageCount` is not a number

Reply:
See other comment on TypeErrors